### PR TITLE
[ingester] support system,user,audit log

### PR DESCRIPTION
**Pull request by lzf575**
### This PR is for:

- Server

### CODE REVIEW

**Review by xiaochaoren1 — APPROVED** [review]

File: server/server.yaml [hunk 1]
```diff
@@ -564,6 +564,22 @@ ingester:
...
+  #  batch-size: 12800   # size of batch writing
+  #  flush-timeout: 5   # timeout of table writing
+
+  ## app log decoder queue count/size
+  #app-log-decoder-queue-count: 2
+  #app-log-decoder-queue-size: 16384
+
+  ## app log database data retention time(unit: hour)
+  ## Note: This configuration is only valid when DeepFlow is run for the first time or the ClickHouse tables have not yet been created
+  #app-log-ttl-hour: 168
```
Comment (sharang) [inline server/server.yaml:581]: application log

File: server/patch/clickhouse-go/write_improve.patch [hunk 2]
```diff
@@ -371,13 +371,13 @@ index 6994106..5951410 100644
...
-+	col.vi = make(map[uint16]string)
++	// col.iv = make(map[string]uint16)
++	// col.vi = make(map[uint16]string)
 +}
 +func (col *Enum8) Reset() {
 +	col.values.data = col.values.data[:0]
-+	col.iv = make(map[string]uint8)
-+	col.vi = make(map[uint8]string)
++	// col.iv = make(map[string]uint8)
++	// col.vi = make(map[uint8]string)
```
Comment (sharang) [inline server/patch/clickhouse-go/write_improve.patch:380]: fixme

File: server/libs/zerodoc/pb/metric.proto [hunk 3]
```diff
@@ -0,0 +1,262 @@
+syntax = "proto3";
```
Comment (sharang) [inline server/libs/zerodoc/pb/metric.proto:1]: 删除？

Comment (lzf575) [reply server/libs/zerodoc/pb/metric.proto]: 删除

File: server/server.yaml (hunk 1 above)
Comment (lzf575) [reply server/server.yaml]: ok

File: server/patch/clickhouse-go/write_improve.patch (hunk 2 above)
Comment (lzf575) [reply server/patch/clickhouse-go/write_improve.patch]: ok

**Review by sharang — APPROVED** [review]

**Pull request merged**